Biotech/Keyed/Letters.xml

```diff
@@ -0,0 +1,197 @@
+
+  <!-- EN: This happened because of -->
+  <MentalStateReason_Gene>Это произошло из-за</MentalStateReason_Gene>
+
+  <!-- EN: {0} summoned -->
+  <LetterLabelBossgroupSummoned>{0} призван</LetterLabelBossgroupSummoned>
+  <!-- EN: Attracted by your mech-band signals, an ultra-heavy mech from {0} will arrive soon. Be prepared. -->
+  <LetterBossgroupSummoned>Поймав сигналы вашего механитора, сверхтяжёлый механоид из фракции {0} решил атаковать ваше поселение, приготовтесь.</LetterBossgroupSummoned>
+
+  <!-- EN: {0} arrived -->
+  <LetterLabelBossgroupArrived>{0} прибыл</LetterLabelBossgroupArrived>
+  <!-- EN: An ultra-heavy mech from {0} has come to eradicate you, along with escorts.\n\n{1}\n\nThe assault group is composed of the following {3}:\n\n{4}\n\nThey will prepare for a while before attacking. -->
+  <LetterBossgroupArrived>Сверхтяжёлый механоид, из фракции {0}, прибыл вместе со своей группой чтобы уничтожить вас.\n\n{1}\n\nГруппа состоит из следующих {lookup: {3}; Case; 1}:\n\n{4}\n\nУ них уйдёт какое-то время на подготовку перед атакой.</LetterBossgroupArrived>
+
+  <!-- EN: Summon {LEADER_label} available -->
+  <LetterLabelCommsConsoleSpawned>Доступен призыв {LEADER_label}</LetterLabelCommsConsoleSpawned>
+  <!-- EN: A mechanitor can use the {PARENT_label} to summon a dangerous and hostile {LEADER_label} mechanoid. Be prepared, {LEADER_label} mechanoids are extremely powerful.\n\nIf you defeat the {LEADER_label}, it will drop the following items:\n\n{REWARDSLIST} -->
+  <LetterCommsConsoleSpawned>Механитор может призвать враждебного механоида {LEADER_label} с помощью {PARENT_label}. Приготовтесь, {LEADER_label} очень опасен.\n\nЕсли вы сумеете уничтожить {LEADER_label}, то получите следующие предметы:\n\n{REWARDSLIST}</LetterCommsConsoleSpawned>
+
+  <!-- EN: Summon {LEADER_label} possible -->
+  <LetterLabelBossgroupCallerUnlocked>Доступен призыв {LEADER_label}</LetterLabelBossgroupCallerUnlocked>
+  <!-- EN: You can now build a {PARENT_label} which a mechanitor can use to summon a {LEADER_label}. If you defeat the {LEADER_label}, it will drop the following items:\n\n{REWARDSLIST} -->
+  <LetterBossgroupCallerUnlocked>Теперь вы можете построить {PARENT_label}, с помощью которого ваш механитор сможет призвать {LEADER_label}. Если вы сумеете уничтожить {LEADER_label}, то получите следующие предметы:\n\n{REWARDSLIST}</LetterBossgroupCallerUnlocked>
```

Comment:
```suggestion
  <LetterLabelCommsConsoleSpawned>Доступен призыв {lookup: {LEADER_label}; Case; 1}</LetterLabelCommsConsoleSpawned>
  <!-- EN: A mechanitor can use the {PARENT_label} to summon a dangerous and hostile {LEADER_label} mechanoid. Be prepared, {LEADER_label} mechanoids are extremely powerful.\n\nIf you defeat the {LEADER_label}, it will drop the following items:\n\n{REWARDSLIST} -->
  <LetterCommsConsoleSpawned>Механитор может призвать враждебного {lookup: {LEADER_label}; Case; 3} с помощью {lookup: {PARENT_label}; Case; 1}. Приготовтесь, этот механоид враждебен и очень опасен.\n\nЕсли вы сумеете уничтожить {lookup: {LEADER_label}; Case; 3}, то получите следующие предметы:\n\n{REWARDSLIST}</LetterCommsConsoleSpawned>

  <!-- EN: Summon {LEADER_label} possible -->
  <LetterLabelBossgroupCallerUnlocked>Доступен призыв {lookup: {LEADER_label}; Case; 1}</LetterLabelBossgroupCallerUnlocked>
  <!-- EN: You can now build a {PARENT_label} which a mechanitor can use to summon a {LEADER_label}. If you defeat the {LEADER_label}, it will drop the following items:\n\n{REWARDSLIST} -->
  <LetterBossgroupCallerUnlocked>Теперь вы можете построить {lookup: {PARENT_label}; Case; 3}, с помощью которого ваш механитор сможет призвать {lookup: {LEADER_label}; Case; 3}. Если вы сумеете уничтожить этого враждебного механоида, то получите следующие предметы:\n\n{REWARDSLIST}</LetterBossgroupCallerUnlocked>
```